statsmodels/discrete/discrete_model.py

```diff
@@ -502,22 +559,66 @@ def _derivative_exog(self, params, exog=None, transform='dydx',
 class MultinomialModel(BinaryModel):
 
     def _handle_data(self, endog, exog, missing, hasconst, **kwargs):
+        reference = kwargs.pop('reference', 0)
```

Comment:
Can we just add `reference=0` to the signature?  Pushing back against the `**kwargs` pattern.